ccd-definition/AuthorisationCaseField.json

```diff
@@ -566,6 +573,13 @@
     "UserRole": "caseworker-publiclaw-courtadmin",
     "CRUD": "CRU"
   },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "CaseFieldID": "applicant",
+    "UserRole": "caseworker-publiclaw-courtadmin",
+    "CRUD": "CRU"
```

Comment:
permissions for courtadmin - applicant need to be reverted back to R

Reply:
👍